src/main/cpp/subsys/SwerveChassis.cpp

```diff
@@ -491,17 +509,44 @@ void SwerveChassis::AdjustRotToPointTowardGoal
     Logger::GetLogger()->ToNtTable(string("Chassis Heading"), string("Field Pos for Toward Goal: TargetAngle(Degrees)"), targetAngle.to<double>());
 
     if (m_limelight != nullptr && m_limelight->HasTarget())
-    {
-        rot += (m_limelight->GetTargetHorizontalOffset())/1_s*kPGoalHeadingControl;
-    }
-    else if (DriverStation::IsAutonomousEnabled())
-    {
-        rot -= CalcHeadingCorrection(targetAngle, kPAutonGoalHeadingControl);;
+    { 
+        if ( abs(error) > 30)
+        {
+            deltaX = (myPose.X()- targetPose.X()).to<double>();
+            deltaY = (myPose.Y()- targetPose.Y()).to<double>();
+            xSpeed -= units::length::meter_t(deltaX)/1_s*kPDistance; 
+            ySpeed -= units::length::meter_t(deltaY)/1_s*kPDistance; 

```

Comment:
This should probably be a separate method.   It works fine for what we need in teleop drive to shooting position, but embedding the driving into the pointing method means we are effecting everything.  If we had a method that was DriveToShootingPosition, it could call AdjustRotToPointTowardGoal and then add in the driving updates including locking the wheels.    Then the orientation gets a new option that goes down this branch.    Then auton could choose this method (wants to return to shooting location) vs. the existing option when it wants to just point toward the goal while driving a path.

Reply:
Agreed, please see first comment and we can talk through this more later. Call me if you want to talk through it before robotics. I am available